packages/mask/package.json

```diff
@@ -28,6 +28,7 @@
     "@masknet/public-api": "workspace:*",
     "@masknet/sdk": "workspace:*",
     "@masknet/shared": "workspace:*",
+    "@masknet/utils": "workspace:*",
```

Comment:
Yet another package for utilities.
 
+ @dimensiondev/kit
+ @masknet/shared
+ @masknet/shared-base

Reply:
I update description  to answer this:

> We have a @masknet/share-base, but it depends on @masknet/web3-shared-evm which is a business related module.
Our codebase really needs a module to hold all context less utility code.

Reply:
Merge utilities from @dimensiondev/kit into this?

Reply:
> We have a @masknet/share-base, but it depends on @masknet/web3-shared-evm which is a business related module.

Actually, I intended to make `@masknet/shared-base` to be context-less.

> @masknet/shared

This package is not for utilities. It's for shared code between dashboard and main extension